resources/authgear/templates/en/web/authflowv2/signup.html

```diff
@@ -82,6 +89,11 @@ <h2 class="screen-description">
             data-retain-form-form-name-param="text"
           >
         {{ end }}
+        {{ if $display_error }}
+          <ul class="input__error-message mt-2">
```

Comment:
1. We should use gap instead of `mt-2`. There are some other places using margin-top as well. We should correct those (Maybe in another issue)
2. By using gap, the usage should become

```html
<div class="input__container">
  <input class="input">
  <ul class="input__error-message">{{ template "authflowv2/__error.html" $ctx }}</ul>
</div>
```

Reply:
I think it is we cannot replace all margin-top with gap, because some elements are not having equal spacing, but for text field we can use gap here.